tflite2onnx/op/padding.py

```diff
@@ -34,6 +35,9 @@ def parse(self):
         opcode = self.model.OperatorCodes(op.OpcodeIndex()).BuiltinCode()
         assert(opcode in self.TypeMapping)
 
+        if opcode is tflite.BuiltinOperator.MIRROR_PAD:
+            self.attrs['mode'] = 'reflect'
```

Comment:
Can we make this to something like below? We prefer to setup `mode` here explicitly.

```py
if opcode is MIRROR_PAD:
    self.attrs['mode'] = 'reflect'
else:
    self.attrs['mode'] = 'constant'
```

Reply:
https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R43-R46